testsuite/MDAnalysisTests/coordinates/test_chemfiles.py

```
@@ -45,7 +45,7 @@ def test_version_check(self):
         import chemfiles
 
         actual_version = chemfiles.__version__
-        chemfiles.__version__ = "0.8.3"
+        chemfiles.__version__ = "0.9.3"
         assert not check_chemfiles_version()
 
         with pytest.raises(RuntimeError, match="Please install Chemfiles > 0.9"):
```

matches need updating here?

---

Ok the failures are specifically due to this test. The test fails early on the first `pytest.raises` so the version doesn't get set back to the correct value.

I have put a PR against your feature branch with a monkeypatch rewrite that should avoid this issue.